modules/serverless-tls-creating-selfsigned-cluster-issuer-eventing.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * /serverless/Eventing/serverless-config-tls-encryption-eventing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-tls-creating-selfsigned-cluster-issuer-eventing_{context}"]
+= Creating a `SelfSigned` `ClusterIssuer` resource for Eventing
+
+`ClusterIssuers` are Kubernetes resources that represent certificate authorities (CAs) capable of generating signed certificates by honoring certificate signing requests. All cert-manager certificates require a referenced issuer in a ready condition to attempt to honor the request. For more details, see link:cert-manager.io/docs/concepts/issuer/[Issuer].
+
+[IMPORTANT]
+====
+For the simplicity, we will use a `SelfSigned` issuer as the root certificate. For more details about `SelfSigned` issuer implications and limitations, see link: https://cert-manager.io/docs/configuration/selfsigned/[SelfSigned issuers]. If you are using a custom public key infrastructure (PKI), you must configure it so its privately signed CA certificates are recognized across the cluster. For more details about cert-manager, see link:https://cert-manager.io/docs/configuration/ca/[certificate authorities (CAs)]. You can use any other issuer that is usable for cluster-local services.
```

Comment:
```suggestion
For the simplicity, we will use a `SelfSigned` issuer as the root certificate authority. For more details about `SelfSigned` issuer implications and limitations, see link: https://cert-manager.io/docs/configuration/selfsigned/[SelfSigned issuers]. If you are using a custom public key infrastructure (PKI), you must configure it so its privately signed CA certificates are recognized across the cluster. For more details about cert-manager, see link:https://cert-manager.io/docs/configuration/ca/[certificate authorities (CAs)]. You can use any other issuer that is usable for cluster-local services.
```

Reply:
^^^ Technically, the issuer is not the "certificate" itself but a certificate authority that issues the certificate.

Reply:
Okay! Noted!